gse-afs-ext-base/src/main/java/com/powsybl/gse/afs/ext/base/ContingencyStoreEditor.java

```diff
@@ -130,13 +131,29 @@ private ContingencyElement createElement(EquipmentInfo equipmentInfo) {
 
             case "LINE":
             case "TWO_WINDINGS_TRANSFORMER":
+            case "BRANCH":
```

Comment:
It’s weird to have Line, TWT and branch in the same switch case as Branch is the base class of the two others